CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+0.5.0 / 2022-02-01
+===================
+Note that this is a full changelog relative to `0.4.4` version. Changelog relative to `0.5.0-RC2` can be found below.
```

Comment:
```suggestion
Note that this is a full changelog relative to `0.4.4` version. Changelog relative to `0.5.0-RC2` can be found at the end.
```

Reply:
"at the end" can be perceived as "at the end of the file"